app/server/lib/ICreate.ts

```diff
@@ -86,7 +88,7 @@ export function makeSimpleCreator(opts: {
   getSandboxVariants?: () => Record<string, SpawnFn>,
   createInstallAdmin?: (dbManager: HomeDBManager) => Promise<InstallAdmin>,
 }): ICreate {
-  const {deploymentType, sessionSecret, storage, notifier, billing, telemetry} = opts;
+  const {deploymentType, storage, notifier, billing, telemetry} = opts;
```

Comment:
I think it is better to leave the parameter here. Otherwise we are breaking the public API of this method for all other external contexts. (Probably all maintain by us, but still).

I'd just add another `|| DEFAULT....` in line 121, at the end.

I think you have access to `saas` repo? There we actually pass this variable and use different `default` value.

Reply:
Okay, I'll bring back the API to override the session secret.